modules/administration-guide/pages/upgrading-che-7-41-on-openshift.adoc

```diff
@@ -1,130 +1,10 @@
-export PRODUCT_OLM_PACKAGE={prod-stable-channel-package}
-export PRODUCT_OLM_STARTING_CSV={prod-stable-channel-starting-csv}
-export PRE_MIGRATION_PRODUCT_OPERATOR_NAMESPACE={prod-namespace} # <2>
-export PRE_MIGRATION_PRODUCT_SHORT_ID={pre-migration-prod-id-short}
-export PRE_MIGRATION_PRODUCT_DEPLOYMENT_NAME={pre-migration-prod-deployment}
-export PRE_MIGRATION_PRODUCT_OPERATOR_NAME={pre-migration-prod-operator}
-export PRE_MIGRATION_PRODUCT_CHE_CLUSTER_CR_NAME={pre-migration-prod-checluster}
-export PRE_MIGRATION_PRODUCT_IDENTITY_PROVIDER_DEPLOYMENT_NAME={identity-provider-id}
-export PRE_MIGRATION_PRODUCT_SUBSCRIPTION_NAME={pre-migration-prod-subscription}
-----
-<1> `{prod-namespace}` or another {orch-namespace} where {prod-prev-short} was previously installed.
-<2> `{prod-namespace}` or another {orch-namespace} where {prod-prev-short} was previously installed.
-
-. Run `chmod` on the downloaded scripts:
-+
-[source,terminal]
-----
-$ chmod +x ./1-prepare.sh ./2-migrate.sh ./3-subscribe.sh ./4-wait.sh
-----
-
-. Run the migration scripts:
-+
-[source,terminal]
-----
-$ ./1-prepare.sh && ./2-migrate.sh && ./3-subscribe.sh && ./4-wait.sh
-----
-
-.Verification
-
-* Open {prod-short} dashboard and log in.
-
-[IMPORTANT]
-====
-If migration fails, use the following steps to roll back to the pre-migration {prod-prev-short} state.
-
-. Set the environment variables to use in the migration scripts:
-+
-[source,bash,subs="+attributes"]
-----
-export INSTALLATION_NAMESPACE={prod-namespace} # <1>
-export PRODUCT_ID={prod-id}
-export PRODUCT_SHORT_ID={prod-id-short}
-export PRODUCT_DEPLOYMENT_NAME={prod-deployment}
-export PRE_MIGRATION_PRODUCT_OPERATOR_NAMESPACE={prod-namespace} # <2>
-export PRE_MIGRATION_PRODUCT_DEPLOYMENT_NAME={pre-migration-prod-deployment}
-export PRE_MIGRATION_PRODUCT_SUBSCRIPTION_NAME={pre-migration-prod-subscription}
-export PRE_MIGRATION_PRODUCT_CHE_CLUSTER_CR_NAME={pre-migration-prod-checluster}
-export PRE_MIGRATION_PRODUCT_OPERATOR_NAME={pre-migration-prod-operator}
-export PRE_MIGRATION_PRODUCT_OLM_PACKAGE={pre-migration-prod-package}
-export PRE_MIGRATION_PRODUCT_OLM_CHANNEL={pre-migration-prod-channel}
-export PRE_MIGRATION_PRODUCT_OLM_CATALOG_SOURCE={pre-migration-prod-catalog-source}
-export PRE_MIGRATION_PRODUCT_OLM_STARTING_CSV={pre-migration-prod-starting-csv}
-----
-<1> `{prod-namespace}` or another {orch-namespace} where {prod-prev-short} was previously installed.
-<2> `{prod-namespace}` or another {orch-namespace} where {prod-prev-short} was previously installed.
-
-. Download the xref:attachment$migration/rollback.sh[rollback.sh] script.
-
-. Run `chmod` on the `rollback.sh` script:
-+
-[source,terminal]
-----
-$ chmod +x ./rollback.sh
-----
-
-. Run the `rollback.sh` script.
-+
-[source,terminal]
-----
-$ ./rollback.sh
-----
-
-
-.Verification
-
-* Open {prod-prev-short} dashboard and log in.
-
-====
+include::example$snip_{project-context}-upgrading-che-7-41-on-openshift.adoc[]
```

Comment:
I see now. Did you validate that this include statement works?

Reply:
Yes. I checked it when generating a preview. Thank you.